src/System.Private.CoreLib/shared/System/SpanHelpers.Char.cs

```diff
@@ -81,6 +81,96 @@ public static unsafe int SequenceCompareTo(ref char first, int firstLength, ref
             return lengthDelta;
         }
 
+        // Adapted from IndexOf(...)
+        public static unsafe bool Contains(ref char searchSpace, char value, int length)
+        {
+            Debug.Assert(length >= 0);
+
+            if (length == 0) return false;
+
+            fixed (char* pChars = &searchSpace)
+            {
+                char* pCh = pChars;
+                char* pEndCh = pCh + length;
+
+#if !netstandard11
```

Comment:
These `#if !netstandard11` ifdefs are no longer required. They can be deleted from all files that you are touching (the code that's inside them should stay).